pinot-broker/src/main/java/org/apache/pinot/broker/queryquota/HelixExternalViewBasedQueryQuotaManager.java

```diff
@@ -207,21 +201,50 @@ private void createOrUpdateRateLimiter(String tableNameWithType, ExternalView br
           "Rate limiter for table: {} has been initialized. Overall rate: {}. Per-broker rate: {}. Number of online broker instances: {}. Table config stat version: {}",
           tableNameWithType, overallRate, perBrokerRate, onlineCount, stat.getVersion());
     } else {
-      queryQuotaEntity.getRateLimiter().setRate(perBrokerRate);
+      RateLimiter rateLimiter = queryQuotaEntity.getRateLimiter();
```

Comment:
Am I understanding right, that this is the case when QPS quota gets added to a table (i.e. queroQuotaEntity already exists but rate limiter does not). If so, adding a comment here will be nice.